commcare_connect/templates/tailwind/components/badges/badge_md.html

```diff
@@ -0,0 +1,5 @@
+<span
+  class="badge inline-block py-1 px-4 mx-1 text-nowrap rounded-3xl text-xs bg-{{ bg_color|default:'[#f1f5f9]' }} text-{{text_color|default:'white' }}"
```

Comment:
Should we define these colors in the tailwind config and use them using varname?